examples/macvlan-conf.yml

```diff
@@ -1,5 +1,5 @@
-apiVersion: "kubernetes.cni.cncf.io/v1"
-kind: Network
+apiVersion: "k8s.cni.cncf.io/v1"
```

Comment:
I think we have to update `k8s.v1.cni.cncf.io/networks` in all  the pod examples.